### PR TITLE
Added test to get info of a specific share id

### PR DESCRIPTION
**Pull request by SergioBertolinSG**

### CODE REVIEW

File: build/integration/features/sharing-v1.feature [hunk 1]
```diff
@@ -108,6 +108,16 @@ Feature: sharing
...
 
+  Scenario: getting share info of a share
+    Given user "user0" exists
+    And user "user1" exists
+    And file "textfile0.txt" from user "user0" is shared with user "user1"
+    And As an "user0"
+    When Getting info of last share
+    Then the OCS status code should be "100"
+    And the HTTP status code should be "200"
+    And User "user1" should be included in the response
```
Comment (DeepDiver1975) [inline build/integration/features/sharing-v1.feature]: I'd prefer to have a more explicit validation here for all xml elements.
We can even use PyString - http://docs.behat.org/en/v2.5/guides/1.gherkin.html#pystrings - to specific the explicit xml response.

@SergioBertolinSG please let me know if you need support

Comment (rullzer) [inline build/integration/features/sharing-v1.feature]: Yes please validate all elements currently there. So we know if we break the API. Now for a lot of stuff you can probably only check if they are there. Or you need to have database access in the process.

Comment (SergioBertolinSG) [inline build/integration/features/sharing-v1.feature]: Ok, I'll modify this to check all elements.

**Comment by SergioBertolinSG** [comment]
I've chosen TableNode instead of PyString, suits better to check the fields.
There are four fields which are different everytime a share is created they are:
storage, file_parent, file_source, item_source. 
I don't know what they are used for, but the content changes. 
Do we want to check them?

**Comment by rullzer** [comment]
>I've chosen TableNode instead of PyString, suits better to check the fields.
There are four fields which are different everytime a share is created they are:
storage, file_parent, file_source, item_source.
I don't know what they are used for, but the content changes.
Do we want to check them?

Yeah that is likely to happen. Just make sure they are set in the return for now.

**Comment by SergioBertolinSG** [comment]
I've added all remaining fields just checking if they have numeric values.

**Comment by rullzer** [comment]
@SergioBertolinSG awesome thanks!

:+1:

**Comment by DeepDiver1975** [comment]
please squash

**Comment by rullzer** [comment]
Could you for a next PR do this for link shares? Since they have different files (token/url, possibly expiration).

**Comment by DeepDiver1975** [comment]
> please squash

done :speak_no_evil:

**Comment by SergioBertolinSG** [comment]
@rullzer sure.

**Comment by SergioBertolinSG** [comment]
merge?

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**